HungarianAlgorithm/Application/FileWriter.cs

```diff
@@ -18,6 +20,30 @@ public static void WriteToOutputFile(string path, Solution solution)
                 Console.WriteLine(e.Message);
                 Environment.Exit(1);
             }
+
+            Console.WriteLine("Optimal solution saved.");
+        }
+        
+        public static void GenerateInputFile(string path, int n, int k, int seed = 0)
+        {
+            Console.WriteLine("Generating new file...");
+
+            try
+            {
+                ProblemInstance problemInstance = ProgramUtils.GenerateProblemInstance(n, k, seed);
```

Comment:
`FileWriter` zajmuje się generowaniem `Problem`ów?

Reply:
`FileWriter` zawsze generuje problemy :) Jak teraz o tym myślę, możemy przerobić to tak, żeby to `ProgramUtils.GenerateProblemInstance` wołało `FileWriter` w celu zapisania wygenerowanej instancji i to ma więcej sensu

Reply:
Tylko wtedy będzie to samo tylko że w drugą stronę - `GenerateProblemInstance` będzie miało 2 odpowiedzialności: generowanie oraz zapisywanie do pliku. Może lepiej będzie w `Main`ie w jednej linii generować instancję, a w drugiej zapisywać do pliku?